src/transaction.ts

```diff
@@ -65,12 +65,18 @@ export interface Statement {
   types?: Type | {[param: string]: Value};
 }
 
+export interface TxnRequestOptions {
```

Comment:
Yeah finding a good name for this is hard. I was half wondering if we could just add these new fields to the existing `RequestOptions`. But I don't know how that affects the types that extend from `RequestOptions`.

Reply:
I think we'll go with using `IRequestOptions` with Pick/Omit so we won't need this anymore.